test/e2e/e2e.go

```diff
@@ -584,15 +581,28 @@ var _ = ginkgo.Describe("e2e control plane", func() {
 			framework.Failf("Unable to connect/talk to the internet: %v", err)
 		}
 
-		masterPods, err := f.ClientSet.CoreV1().Pods("ovn-kubernetes").List(context.Background(), metav1.ListOptions{
-			LabelSelector: "name=ovnkube-master",
+		multiZones := isInterconnectEnabled()
```

Comment:
nit: this doesnt necessarily mean there are multiple zones, it just means IC is enabled? might be good to just rename the variable to isICEnabled